runtime/crab/src/pallets/staking.rs

```diff
@@ -12,24 +12,24 @@ use darwinia_staking::{Config, EraIndex};
 #[cfg(feature = "dev")]
 frame_support::parameter_types! {
 	pub const BondingDurationInEra: BlockNumber = 2;
-	pub const BondingDurationInBlockNumber: BlockNumber = 2 * SESSIONS_PER_ERA * BLOCKS_PER_SESSION;
+	pub const BondingDurationInBlockNumber: BlockNumber = 2 * CRAB_SESSIONS_PER_ERA * CRAB_BLOCKS_PER_SESSION;
 	pub const SlashDeferDuration: EraIndex = 1;
 }
 #[cfg(not(feature = "dev"))]
 frame_support::parameter_types! {
 	pub const BondingDurationInEra: EraIndex = 14 * DAYS
-		/ (SESSIONS_PER_ERA as BlockNumber * BLOCKS_PER_SESSION);
+		/ (CRAB_SESSIONS_PER_ERA as BlockNumber * CRAB_BLOCKS_PER_SESSION);
 	pub const BondingDurationInBlockNumber: BlockNumber = 14 * DAYS;
 	// slightly less than 14 days.
 	pub const SlashDeferDuration: EraIndex = 14 * DAYS
-		/ (SESSIONS_PER_ERA as BlockNumber * BLOCKS_PER_SESSION) - 1;
+		/ (CRAB_SESSIONS_PER_ERA as BlockNumber * CRAB_BLOCKS_PER_SESSION) - 1;
 }
 frame_support::parameter_types! {
 	pub const StakingPalletId: PalletId = PalletId(*b"da/staki");
-	pub const SessionsPerEra: SessionIndex = SESSIONS_PER_ERA;
+	pub const SessionsPerEra: SessionIndex = CRAB_SESSIONS_PER_ERA;
 	// quarter of the last session will be for election.
 	pub const MaxNominatorRewardedPerValidator: u32 = 64;
-	pub const Cap: Balance = CAP;
+	pub const Cap: Balance = RING_HARD_CAP;
```

Comment:
We have already renamed the token, no `Ring` in the Crab network now. Please change it to `Crab_HARD_CAP`

Reply:
> We have already renamed the token, no `Ring` in the Crab network now. Please change it to `Crab_HARD_CAP`

I would like to keep this.
Just like why we use RingBalance instead of CrabBalance in the pallet.